### DISCUSSION TITLE
Team HTTP API: add uid field

### DISCUSSION THREAD

**Discussion by weisdd** [grafana/grafana#55595] [Platform]
As far as I know, there's an effort to give users more control over identifiers used for various resources in Grafana API. That's definitely something useful for projects like https://github.com/grafana-operator/grafana-operator (btw, next version of it will rely on https://github.com/grafana/grafana-api-golang-client), because it allows us to provision resources under the same uid across many instances of grafana, removing the need for complex internal mappings.

It would be great to expand Team HTTP API to offer `uid` field like it's done for dashboards and many other resources.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
